### ISSUE TITLE
Clarification regarding 'OBI:specimen identifier' and 'OMIABIS:specimen ID'

### ISSUE THREAD

**Issue by PhiBabs935** [obi-ontology/obi#1246]
Hi,

I have a question regarding OBI's 'specimen identifier' and OMIABIS's 'specimen ID'.  I ask because the OBI term is listed as having the alternative term 'specimen ID'. 

'specimen identifier' is classified as a 'centrally registered identifier symbol', whereas the OMIABIS term is classified as an 'identifier' and has an axiom specifying: 'has part' some 'centrally registered identifier symbol'. I guess my question is: is there any inconsistency here between the OMIABIS and OBI representation? 

-Shane

**Comment by mbrochhausen** [comment]
I would not necessarily say that there is an inconsistency, but this is something that OBI and OBIB need to sort out. 
In my understanding what the OBI definition says is that, if I have a central registry called BR (short for biobank registry), "BR-12345" is not a 'specimen identifier', but "12345". While that is counterintuitive to me, I am sure that when OBIB decided for that design we did have a reason.
OMIABIS (and thus OBIB) does not take stance on that. "BR-12345" is a specimen identifier and "12345" can be a specimen identifier, too. If I hand of an identifier to a colleague and add "-MB" to signify provenance, it is still an identifier. I am open to discuss and I would be interested to learn the background of the OBI modeling.
Oh, and obviously OMIABIS uses identifier as a superclass, which might not have existed in OBI at the time 'specimen identifier' was created.
Needless to say that the label from OMIABIS does not comply with OBO Foundry rules since it uses an acronym, but that can be easily changed.

**Comment by zhengj2007** [comment]
Based on the definitions, the OBI's 'specimen identifier' seems a subclass of OMIABIS's 'specimen ID'. I think specimen ID may not necessary to be a CRID symbol. I would like to have a more general specimen ID in OBI and assign current OBI_0001616 as its subclass and relabel it. 

@cstoeckert, How do you think about it? We added OBI_0001616 for NIAID GSCID-BRC work.

**Comment by mbrochhausen** [comment]
I think that is a good solution, except that I would also want to stress
that the new specimen ID class needs to be a subclass to "identifier".

Best,
Mathias

**Comment by zhengj2007** [comment]
new specimen ID class needs to be a subclass to "identifier"
- It sounds good to me.

**Comment by linikujp** [comment]
Is there an "identifier" in OBI?
The label of OBI:"specimen identifier" and OMIABIS's 'specimen ID' will be very confusing for the future users who do not know about this conversation. People use ID and identifier exchangeably.
Will it be possible to the labels? -Mathias, since you have mentioned that specimen ID label can be changed. Why not you propose one?
For me, one is a numeric string, the other is a combination of prefix and numeric string.

**Comment by PhiBabs935** [comment]
'identifier' is imported in OBI from IAO.

"I would like to have a more general specimen ID in OBI and assign current OBI_0001616 as its subclass and relabel it." @zhengj2007 
Is the idea then to add a new more general specimen ID term to OBI as a replacement for OMIABIS's 'specimen ID'? [with label adjustments]

Connected to my original, topic: OBIB currently imports the Influenza Ontology term 'specimen barcode' as a subclass of OMIABIS's 'specimen ID'. I am currently working on an updated/cleaned up version of FLU for the IDO group, and am trying to move a variety of FLU terms that are more general than influenza. Can 'specimen barcode' get moved to OBI or OBIB along with these changes?

**Comment by mbrochhausen** [comment]
In line with Jie I propose to import 'specimen id' from OBIB, rename it
specimen identifier. Retain OBI's 'specimen identifier' as a subclass and
rename that, too (Jie: If I am misrepresenting your thoughts, please
correct me).
Please note that neither of the definition says anything about whether
those are numeric strings. It is totally perceivable that a CRID symbol is
made up from letters.

Best,
Mathias

**Comment by PhiBabs935** [comment]
Thanks. For the import of 'specimen ID' from OBIB, would the 'has part' some 'centrally registered identifier symbol' be retained? Given that 'specimen barcode' is a subclass in OBIB, are the numbers on the barcode the CRID symbol part?

**Comment by mbrochhausen** [comment]
Yes, in my mind the has part statement would be retained. Assuming that the numbers on the barcode can be used to identify the specimen, they are CRID symbols.

**Comment by cstoeckert** [comment]
Discussed on Feb 8, 2021. Agree these belong in OBI from FLU and OBIB.
- specimen barcode
    - vacutainer specimen barcode
    - blood plasma specimen barcode
    - cryovial specimen barcode
    - cell pellet specimen barcode
    - specimen randomization key ID
